config/branch_protector/rules.yaml

```diff
@@ -0,0 +1,45 @@
+# Copyright 2020 The Knative Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+branch-protection:
+  orgs:
+    knative:
+      # Protect all branches in knative
+      protect: true
+      required_status_checks:
+        contexts:
+        - cla/google
+        - tide
+      # Enforce all configured restrictions above for administrators.
+      enforce_admins: true
```

Comment:
Can we canary this in knative-sandbox, and then follow up with knative?

Reply:
Sure, I have removed knative here.